grid/project-management/tasks/maps/asana.suma

```diff
@@ -22,10 +18,193 @@ map CreateTask {
     }
     
     response 201 "application/json" {
-      return map result {
-        id = body.data.gid
-        url = body.data.permalink_url
+      data = {
+        id: body.data.gid,
+        title: body.data.name,
+        description: body.data.notes,
+        projects: body.data.projects.map(project => project.gid),
+        parent: body.data.parent,
+        assignee: body.data.assignee.gid,
+        url: body.data.permalink_url
+      }
+
+      return map result data
+    }
+
+    response {
+      error = call MapAsanaError(
+        statusCode = statusCode,
+        error = body,
+        headers = headers
+      )
+      
+      map error error
+    }
+  }
+}
+
+map ListTasks {
+  http GET "/projects/{input.id}/tasks" {
+    security "personalAccessToken"
+    
+    response 200 "application/json" {
+      data = body.data.map(task => ({
+        id: task.gid,
+        title: task.name,
+      }))
+    
+      return map result data
+    }
+
+    response {
+      error = call MapAsanaError(
+        statusCode = statusCode,
+        error = body,
+        headers = headers
+      )
+      
+      map error error
+    }
+  }
+}
+
+map ReadTask {
+  http GET "/tasks/{input.id}" {
+    security "personalAccessToken"
+    
+    response 200 "application/json" {
+      data = {
+        id: body.data.gid,
+        title: body.data.name,
+        description: body.data.notes,
+        projects: body.data.projects.map(project => project.gid),
+        parent: body.data.parent,
+        assignee: body.data.assignee.gid,
+        url: body.data.permalink_url
+      }
+      
+      return map result data
+    }
+
+    response {
+      error = call MapAsanaError(
+        statusCode = statusCode,
+        error = body,
+        headers = headers
+      )
+      
+      map error error
+    }
+  }
+}
+
+map UpdateTask {
+  http PUT "/tasks/{input.id}" {
+    security "personalAccessToken"
+    
+    request "application/json" {
+      body {
+        data = {
+          name: input.title,
+          notes: input.description,
+          // projects: [input.project],
```

Comment:
issue: you have commented out code here without any comment, IMO the commented out properties should be updatable